extensions/amp-sona/amp-sona.md

```diff
@@ -0,0 +1,65 @@
+---
+$category@: ads
+formats:
+  - websites
+teaser:
+  text: Displays sona widget.
+---
+
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Can you update this value to 2020? The value should match the year the new file is committed.

Reply:
👍